<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="2" total="3" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="52230" started-at="2018-11-25T18:29:26Z" finished-at="2018-11-25T18:30:18Z">
    <groups>
    </groups>
    <test name="TestWeightWatcher" duration-ms="52230" started-at="2018-11-25T18:29:26Z" finished-at="2018-11-25T18:30:18Z">
      <class name="Test.Tests">
        <test-method status="PASS" signature="setUp(java.lang.String)[pri:0, instance:Test.Tests@26653222]" name="setUp" is-config="true" duration-ms="51079" started-at="2018-11-25T18:29:26Z" finished-at="2018-11-25T18:30:17Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[ch]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="weightWatchers()[pri:0, instance:Test.Tests@26653222]" name="weightWatchers" duration-ms="27" started-at="2018-11-25T18:30:17Z" finished-at="2018-11-25T18:30:17Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at Utilities.Functionalities.typeByXpath(Functionalities.java:10)
at Test.Tests.weightWatchers(Tests.java:16)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- weightWatchers -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:Test.Tests@26653222]" name="tearDown" is-config="true" duration-ms="1056" started-at="2018-11-25T18:30:17Z" finished-at="2018-11-25T18:30:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Test.Tests -->
    </test> <!-- TestWeightWatcher -->
  </suite> <!-- Suite -->
</testng-results>
